x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/string/EndsWith.java

```diff
@@ -86,14 +92,14 @@ public ScriptTemplate asScript() {
     }
     
     protected ScriptTemplate asScriptFrom(ScriptTemplate sourceScript, ScriptTemplate patternScript) {
-        return new ScriptTemplate(format(Locale.ROOT, formatTemplate("{eql}.%s(%s,%s)"),
-                "endsWith",
-                sourceScript.template(),
-                patternScript.template()),
-                paramsBuilder()
-                    .script(sourceScript.params())
-                    .script(patternScript.params())
-                    .build(), dataType());
+        ParamsBuilder params = paramsBuilder();
+
+        String template = formatTemplate("{eql}.endsWith(" + sourceScript.template() + "," + patternScript.template() + ",{})");
```

Comment:
We're not totally consistent in these. Here's something I did for `stringContains`
https://github.com/elastic/elasticsearch/blob/8fb5b566df3703aae22fcd75db3527e256dd44b4/x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/string/StringContains.java#L87-L98

Reply:
Followed Andrei implementation for consistency, can change back. Let me know.

Reply:
Updated PR